torch/csrc/jit/codegen/cuda/codegen.cpp

```diff
@@ -705,6 +706,26 @@ class CudaKernelGenerator : private OptOutConstDispatch {
     }
 
     if (!print_inline_) {
+      if (op_type == UnaryOpType::RandLike) {
```

Comment:
Should we assert that we're not trying to print randlike inline? Or maybe we should just ignore the inline if it's a rand op?

Reply:
assert added